frontend/src/components/ViewTrips/edit-trip-button.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```

Comment:
Is it important to have a dedicated file for the edit trip button vs just including the button in the trip file? Is it used in multiple files? If another file is needed, what are the benefits of individual files vs a buttons file with all buttons in it?

Reply:
There is no advantage to having it defined in `trip.js` or its own individual style but I thought it would be nice for consistency sake to abstract edit trips button since the buttons for delete and view activities were.

I think I will keep it as is for those reasons but if you think it would be more clear to include the button in `trip.js` just let me know and I will make the change.